apps/api/src/app/rate-limiting/usecases/get-api-rate-limit-service-maximum-config/get-api-rate-limit-service-maximum-config.usecase.ts

```diff
@@ -39,7 +39,7 @@ export class GetApiRateLimitServiceMaximumConfig implements OnModuleInit {
     if (previousHash !== newHash) {
       await this.cacheService.set(cacheKey, newHash);
 
-      this.invalidateCache.invalidateQuery({
+      this.invalidateCache.invalidateByKey({
```

Comment:
@djabarovgeorge made this change per the PR feedback: https://github.com/novuhq/novu/pull/4910#discussion_r1408493196